Yams.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name                      = 'Yams'
-  s.version                   = '5.0.6'
+  s.version                   = '5.1.0'
```

Comment:
Version should be updated to 5.2.0

Reply:
Please don’t touch this file. I’ll bump the version when the time comes to cut a release.